<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_main_window</class>
 <widget class="QMainWindow" name="customer_main_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Customer Main Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(5, 130, 202);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="csmainwdw_lbl_heading">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bank Transactions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_getcash">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 100, 148);

border:2px solid rgb(0, 150, 199);
border-radius:20px;
border-color:black;
}
QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(0, 150, 199);
}</string>
    </property>
    <property name="text">
     <string>GET CASH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_returnmain">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>70</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_deposit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(0, 100, 148);

border:2px solid rgb(0, 150, 199);
border-radius:20px;
border-color:black;
}
QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(0, 150, 199);
}</string>
    </property>
    <property name="text">
     <string>DEPOSIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_moneyup">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="csmainwdw_btn_moneydown">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(0, 150, 199);
   border-color: rgb(66, 167, 255);
   border-bottom-color: rgb(255, 255, 255);
   border:2px solid rgb(202, 240, 248);
   border-radius:20px;
   border-color: black;
}

QPushButton:hover {
     background-color: rgb(202, 240, 248);
     border:2px solid rgb(202, 240, 248);
}
    
   </string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="csmainwdw_spinbox_money">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(236, 236, 236);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_resultmessage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>501</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiBold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_CSname_show">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>100</y>
      <width>228</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_CSID_show">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>160</y>
      <width>228</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_balanceshow">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>220</y>
      <width>180</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_CSID_heading">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>182</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Customer ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_CSname_heading">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>228</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Customer Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="csmainwdw_lbl_balanceheading">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>327</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Your current balance:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>csmainwdw_btn_moneyup</sender>
   <signal>clicked(bool)</signal>
   <receiver>csmainwdw_spinbox_money</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csmainwdw_btn_moneydown</sender>
   <signal>clicked(bool)</signal>
   <receiver>csmainwdw_spinbox_money</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
